src/ffis.ts

```diff
@@ -31,4 +38,21 @@ export const INCHIAPI = Library(join(__dirname, "./libinchi.so.1.06.00"), {
   GetStructFromStdINCHI: [refNAPI.types.int, [refNAPI.refType(inchi_InputINCHI), refNAPI.refType(inchi_OutputStruct)]],
   FreeStructFromINCHI: [refNAPI.types.void, [refNAPI.refType(inchi_OutputStruct)]],
   FreeStructFromStdINCHI: [refNAPI.types.void, [refNAPI.refType(inchi_OutputStruct)]],
+  GetINCHI: [refNAPI.types.int, [refNAPI.refType(inchi_Input), refNAPI.refType(inchi_Output)]],
+  GetINCHIEx: [refNAPI.types.int, [refNAPI.refType(inchi_InputEx), refNAPI.refType(inchi_Output)]],
+  FreeINCHI: [refNAPI.types.void, [refNAPI.refType(inchi_Output)]],
+  Free_inchi_Input: [refNAPI.types.void, [refNAPI.refType(inchi_Input)]],
+  // please review the last parmater , it is written as inchi_InputData in inchi ref book but in description it is written as inchi_Input
+  Get_inchi_Input_FromAuxInfo: [
+    refNAPI.types.int,
+    [refNAPI.types.CString, refNAPI.types.int, refNAPI.types.int, refNAPI.refType(inchi_Input)],
```

Comment:
```suggestion
    [refNAPI.types.CString, refNAPI.types.int, refNAPI.types.int, refNAPI.refType(InchiInpData)],
```

The docs say:

> pInchiInp
> should have a valid pointer pInchiInp->pInp to an empty (all members = 0) inchi_Input structure

It makes sense because we do have `pInp` in `InchiInpData`:

```ts
export const InchiInpData = NAPIStructType({
  pInp: refNAPI.refType(inchi_Input),
  bChiral: refNAPI.types.int,
  szErrMsg: NAPIArrayType(refNAPI.types.char, STR_ERR_LEN),
});
```